docs/faststore/docs/reference/icons.mdx

```diff
@@ -79,13 +79,19 @@ You can find more details about the `Icon` component [here](https://developers.v
 
 <IconList>
   <IconComponent icon={<Icon name="Visa" />} name="Visa" />
+  <IconComponent icon={<Icon name="Amex" />} name="Amex" />
+  <IconComponent icon={<Icon name="ApplePay" />} name="ApplePay" />
+  <IconComponent icon={<Icon name="EloCard" />} name="EloCard" />
   <IconComponent icon={<Icon name="Diners" />} name="Diners" />
+  <IconComponent icon={<Icon name="GooglePay" />} name="GooglePay" />
   <IconComponent icon={<Icon name="Mastercard" />} name="Mastercard" />
   <IconComponent icon={<Icon name="EloCard" />} name="EloCard" />
```

Comment:
```suggestion
```

**todo**: Removing because it's duplicates

Reply:
Thanks, @hellofanny